go.mod

```diff
@@ -42,7 +42,6 @@ require (
 replace (
 	github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220414164044-61404de7df1a+incompatible
 	github.com/jessevdk/go-flags => github.com/alexcb/go-flags v0.0.0-20210722203016-f11d7ecb5ee5
-	github.com/moby/buildkit => github.com/earthly/buildkit v0.0.1-0.20220721222420-7a6f9e1ab2a3
-	github.com/tonistiigi/fsutil => github.com/earthly/fsutil v0.0.0-20220719234708-392da7eebeb5
-  github.com/earthly/cloud-api => ../cloud-api
+	github.com/moby/buildkit => github.com/earthly/buildkit v0.0.1-0.20220719221101-47e36164897b
```

Comment:
this is unexpected. Given all of our changes to buildkit and fsutil. I would avoid changing this unless it's related to this PR.

Reply:
I think you may be looking at an old commit. I reverted that here: https://github.com/earthly/earthly/pull/2050/commits/3313a5141b1e5e54a33c7680f774d59537bcf7ef